cmd/fyne/package-common.go

```diff
@@ -0,0 +1,193 @@
+package main
```

Comment:
for this file I would just call it "package.go" we use "xxx-common" or "xxx-other" when it's a fallback to functionality that is otherwise OS specific, but in this case it's the loader that will access the OS code I think?

Reply:
Sure. Just thought it looked better with them all being -something. Will change back :)